.changeset/fluffy-hands-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/switch": patch
+---
+
+Fix active down state specificity
```

Comment:
Could you expound a little on this in a long format? We use this information to populate our Changelogs so we try to include as much as possible to help customers validate this update on their end.